thinkhazard/views/report.py

```diff
@@ -1,21 +1,30 @@
-import itertools
 import geoalchemy2.shape
 
 from pyramid.view import view_config
 from pyramid.httpexceptions import HTTPBadRequest
 
 from sqlalchemy.orm import aliased
-from sqlalchemy import and_
+from sqlalchemy import and_, or_, null
+from sqlalchemy.sql.expression import literal_column
 
+from geoalchemy2 import Geometry
 from geoalchemy2.shape import to_shape
 
+from geoalchemy2.functions import GenericFunction
+
 from ..models import (
     DBSession,
-    AdminLevelType,
     AdministrativeDivision,
     CategoryType,
     HazardCategory,
-    HazardType)
+    HazardType,
+    AdditionalInformation,
+    AdditionalInformationType,)
+
+
+class StSimplify(GenericFunction):
+    name = 'ST_Simplify'
+    type = Geometry
```

Comment:
`ST_Simplify` is already declared by GeoAlchemy. See https://github.com/geoalchemy/geoalchemy2/blob/master/geoalchemy2/functions.py#L317-L319.